build/amis/README.md

```diff
@@ -121,9 +131,49 @@ packer build -var-file base-images-us-east-1.json -var ami_regions='us-west-2'
 
 Connect remotely to an instance created from the image and run the Node Conformance tests using the following commands:
 
+### Initialize a CNI
```

Comment:
When are these tests run?
Does it make sense to add these steps into a script that can be run? That might be a step forward towards some of the test automation that we have open issues for. WDYT?

Reply:
Eventually, yes. Currently these tests are only run to validate a new AMI image, which is still a manual step.